frontend/src/components/UserSettings/ChangePassword.tsx

```diff
@@ -43,7 +43,7 @@ const ChangePassword = () => {
     },
     onError: (err: ApiError) => {
       const errDetail = (err.body as any)?.detail
-      showToast("Something went wrong.", `${errDetail}`, "error")
+      showToast("Something went wrong.", errDetail, "error")
```

Comment:
maybe we can do the same logic as above here too?

Reply:
done!